docs/Configuration.md

```diff
@@ -99,6 +99,9 @@ The following configurations are related to Velox settings.
 | spark.gluten.sql.columnar.backend.velox.orc.scan.enabled             | Enable velox orc scan. If disabled, vanilla spark orc scan will be used.                                                                           | true              |
 | spark.gluten.sql.complexType.scan.fallback.enabled                   | Force fallback for complex type scan, including struct, map, array.                                                                                | true              |
 | spark.gluten.velox.offHeapBroadcastBuildRelation.enabled             | Experimental: If enabled, broadcast build relation will use offheap memory. Otherwise, broadcast build relation will use onheap memory, default value is false |                   |
+| spark.gluten.auto.adjustStageResource.enabled                        | Experimental: If enabled, gluten will try to set the stage resource according to stage execution plan. Only worked when aqe is enabled at the same time!! | false   |
```

Comment:
Only worked when aqe is enabled at the same time!!  ==> NOTE: Only works when aqe is enabled at the same time.